### DISCUSSION TITLE
2021-04-29-General-Meeting

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/Governance-HOLON#67] [Meetings]
## Governance HOLON General Meeting

April 29th, 2021 at 7pm UTC

## Attendees (alphabetical order) 

Dan, Hud, Mo, Sev, Stephen, Troy

## Introductions

Dan, Hud, Mo, Sev, Stephen, Troy

**Comment by stephen-rowan** [comment]
Hey everyone, I have set up a Category for Meetings and started a thread for our general meeting of  -

## April 29th, 2021 at 7pm UTC

I am posting my own suggestions for the meeting content here. Then as a group we can decide what should go in the main Agenda above.
Please post below if you have your own suggestions or alternatively let me know by other means (Discord etc) and I will post for you.

## My Suggestions -

### Introductions : 

- [ ] Gov HOLON to briefly explain what each of us are doing for the benefit of Bruno and Mo.
- [ ] Introduce Bruno and Mo.

**Reply by tttroyyy** [reply]
Brilliant, thank you Stephen

**Comment by stephen-rowan** [comment]
##  Some Meeting Notes :

## Note

Because Gov-HOLON plans to use sourcecred or similar software to track our contributions - from this point on I am going to try to include everything in the GitHub system. 

## Priorities

**Everyone** - After introductions there was a general agreement to focus on the short-term deliverable of a contribution token. Particularly using the [sourcecred platform](https://sourcecred.io/) as a means to incentivize contributions to Discord and GitHub.

## sourcecred set-up 

**Hud, Stephen** to chase sourcecred  to resolve any bugs in building a platform to test on.

A discussion thread to document our setup here  ( https://github.com/NFT-DAO/Governance-HOLON/discussions/80)

Stephen has setup a Project Board for sourcecred  setup (at https://github.com/NFT-DAO/Governance-HOLON/projects/9 ), raised an issue for resolving the bug ( https://github.com/NFT-DAO/Governance-HOLON/issues/77) and set a Milestone Deadline of May 07, 2021 to complete setup (https://github.com/NFT-DAO/Governance-HOLON/milestone/7 ) 

## Introducing Mo

**Mo** was able to bring a fresh perspective to Gov-HOLON planning reminding the group of what are immediate problems, solutions and goals are. Who the actors are in a network, the token types, how to capture value and constraints.

## NFT-DAO's main objective

**Sev** - emphasized that NFT-DAO's main objective was the delivery of the BOXCAR solution.

## Business Planning and the Charter

**Troy, Stephen** - to continue to work together Business Planning (formatting the BP) and outputs (such as the Charter). Stephen : In my view at a minimum Gov-HOLON as a group should provide input into the Charter. 

The Project Board for the NFT-DAO-Charter ( https://github.com/NFT-DAO/Governance-HOLON/projects/4 )
The first section of the Charter "Who We Are/Background" to be completed by Gov-HOLON raised as an Issue (https://github.com/NFT-DAO/Governance-HOLON/issues/69) has a Milestone of May 06, 2021 ( https://github.com/NFT-DAO/Governance-HOLON/milestone/3) 

There is a Project Boards for Business-Plan-Decomposition ( https://github.com/NFT-DAO/Governance-HOLON/projects/3 ) this will be updated and refined by Troy and Stephen to more closely track NFT-DAO Business Planning Issues and Milestones.

There is a discussion thread for Business-Plan-Decomposition ( https://github.com/NFT-DAO/Governance-HOLON/discussions/52 ) - Troy and Stephen to work together on what discussion would be most useful to document here.

## GitHub training

**Sev, Stephen & Troy** - to arrange some time for GitHub training. This will be tailored to Gov-HOLONs immediate needs and support Sev's Business Research work as well as Troy's Business Planning work. More details will be provided under a GitHub Training Discussion Thread ( here https://github.com/NFT-DAO/Governance-HOLON/discussions/79).

Stephen is also hoping to capture some of the training process so that it can be re-used. Also Sev, Stephen & Troy will try to track their training to prepare for contribution metrics.

Stephen has setup a Project Board for GitHUb Training (at https://github.com/NFT-DAO/Governance-HOLON/projects/10 ), raised an issue for setting up training ( https://github.com/NFT-DAO/Governance-HOLON/issues/78) and set a Milestone Deadline of May 27, 2021 to complete setup (https://github.com/NFT-DAO/Governance-HOLON/milestone/8)

**Comment by tttroyyy** [comment]
[Uploading NFT-DAO GovHolon Meeting MINUTES (30-4-21 UTC).pdf…]()
